src/java.base/share/classes/java/security/Security.java

```diff
@@ -74,11 +91,222 @@ private static class ProviderProperty {
         Provider provider;
     }
 
+    private static final class SecPropLoader {
+        private enum LoadingMode {OVERRIDE, APPEND}
+
+        private static final String OVERRIDE_SEC_PROP =
+                "security.overridePropertiesFile";
+
+        private static final String EXTRA_SYS_PROP =
+                "java.security.properties";
+
+        private static Path currentPath;
+
+        private static final Set<Path> activePaths = new HashSet<>();
+
+        static void loadAll() {
+            // first load the master properties file to
+            // determine the value of OVERRIDE_SEC_PROP
+            loadMaster();
+            loadExtra();
+        }
+
+        static boolean isInclude(String key) {
+            return "include".equals(key);
+        }
+
+        private static void loadMaster() {
```

Comment:
We had internally discussed about this. In order to avoid confusion, we decided to align with the terminology already present in `java.security`.

Given that such a change would require a bigger searching effort (e.g. what terminology is used in user guides and/or other public APIs documentation?), we think this should be addressed in its own enhancement.

Reply:
Oh, sorry I didn't realize the `java.security` file also uses it. I remember long long ago we've gone through the JDK codebase on this naming issue. I don't remember if this one was not considered an issue or had been exempted. Anyway, yes we don't need to touch it this time.